socorro/unittest/cron/crontabber_tests_base.py

```diff
@@ -46,7 +48,7 @@ def _setup_config_manager(self, jobs_string, extra_value_source=None):
 
         """
         mock_logging = mock.Mock()
-        required_config = app.CronTabber.get_required_config()
+        required_config = CronTabberApp.get_required_config()
```

Comment:
I went through all the tests and made sure they were testing with Socorro's `CronTabberApp`. So there's a bunch of string replacement along those lines.